src/wallet/wallet.cpp

```diff
@@ -784,6 +784,27 @@ bool CWallet::IsSpentKey(const uint256& hash, unsigned int n) const
     return false;
 }
 
+void CWallet::SetUsedAddressState(WalletBatch& batch, const CTxOut& cout, bool used)
+{
+    AssertLockHeld(cs_wallet);
+    CTxDestination dst;
+    if(ExtractDestination(cout.scriptPubKey, dst)) {
```

Comment:
Please update code format accordingly to developer notes.

Reply:
Updated according to developer notes (added space after `if`), hopefully I haven't missed anything, Thanks!

Reply:
If helpful, you can use Clang in the future to check for some things:

    Use Clang formatting to clean up a file:
    clang-format -i <file>

    To format the last commit with 0 lines of context, call the
    clang-format-diff.py script from root as follows:
    git diff -U0 HEAD~1.. | ./contrib/devtools/clang-format-diff.py -p1 -i -v

Reply:
Thanks @jonatack, that will definitely be useful for me to use in the future!